llarp/config/config.cpp

```diff
@@ -711,6 +711,8 @@ namespace llarp
     // Default, but if we get any upstream (including upstream=, i.e. empty string) we clear it
     constexpr Default DefaultUpstreamDNS{"1.1.1.1"};
     m_upstreamDNS.emplace_back(DefaultUpstreamDNS.val);
+    if (!m_upstreamDNS.back().getPort())
+        m_upstreamDNS.back().setPort(53);
```

Comment:
Did it this way rather than putting :53 in the default value so that the default shown in the config remains 1.1.1.1 rather than 1.1.1.1:53